### DISCUSSION TITLE
Text allingning for skin editor

### DISCUSSION THREAD

**Discussion by mandarinka12051** [ppy/osu#26236] [Ideas] [answered]
I have a lot of text elements in my osu lazer custom skin layout. I find it very annoying to have it alway alling to the center, when i put the player profile picture and the "Autoplay" player name next to it, it looks good. But when i enter the game, my own name "Mandarinka125" is over the profile picture. I'd really appriciate if you'd add an option to all text elements if they should be allinged to the left, center or right. It would make the skin layout customization more customizable.

**Comment by bdach** [comment, marked as answer]
Right click element -> change anchor & origin of element to suit your need.
